include/rocksdb/merge_operator.h

```diff
@@ -222,26 +222,38 @@ class MergeOperator {
   }
 };
 
-// The simpler, associative merge operator.
+// A merge operator that can be implemented as an associative binary
+// operator on values/operands. Implementations must satisfy
```

Comment:
I suggest to use the term `input` when the parameter could be either a `value` or an `operand`:

```
// operator on an input (either a value or a merge operand), and a merge operand. Implementations must satisfy
```